inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/util/TestFixtures.java

```diff
@@ -140,4 +143,23 @@ public KBQualifier buildQualifier(KBStatement kbStatement, KBHandle propertyHand
             vf.createLiteral(value));
         return qualifier;
     }
+
+    /**
+     * Tries to connect to the given endpoint url
+     * @param aEndpointURL the url to check
+     * @param aTimeout a timeout
+     * @return returns true if no IOException is thrown when connecting to the url, false otherwise
+     */
+    public boolean isEndpointAvailable(String aEndpointURL, int aTimeout) {
+        try {
+            URLConnection connection = new URL(aEndpointURL).openConnection();
+            connection.setConnectTimeout(aTimeout);
+            connection.connect();
+            return true;
+        }
+        catch (IOException e) {
+            e.printStackTrace();
```

Comment:
A stack trace is a bit of an overkill here. You can print (or log) a simple message here that the endpoint is not available. But actually, I'd suggest to rename this to `assumeEndpointIsAvailable(...)` and instead of returning false, you just call `assume` at the end using the signature of the method which allows including a message. In that message you can include the URL of the endpoint as well as the reason for failure from the exception message. That way the information why the test was skipped directly ends up having all the details.